### ISSUE TITLE
Missing blank(s) between (echo) command and argument not detected

### ISSUE THREAD

**Issue by wehe00** [koalaman/shellcheck#2311]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash
echo"hello"

```

#### Here's what shellcheck currently says:
No issues detected!

#### Here's what I wanted or expected to see:
Missing blank(s) between (echo) command and argument.

**Comment by koalaman** [comment]
I had a look and found a few cases of this (mostly `echo` in fact), but I was unable to find a nice general pattern with a high signal:noise ratio. Common false positives were indirections like `run"$1"`, and scripts with mismatched quotes where `echo "foo"` was actually `"echo "foo"..."` and already detected. Several of the true positives like `echo"hello."` were also already detected by the more general "command ends with dot" checks. I'll leave this one to the shell's fairly clear error message. 

Thanks for the suggestion!
